packages/plugins/experimental/plugin-wnfs/src/components/FileMain.tsx

```diff
@@ -0,0 +1,23 @@
+//
```

Comment:
is it possible to create a unit test or a react story to demo upload and preview of images? e.g., look at TextEdtitor.stories.tsx (low-level) or mermaid.stories.tsx (which is another plugin).

Reply:
Sure yeah, I'll add that to this PR. I assume without involving the blob service right?

Reply:
@richburdon Added one.
![Screenshot 2024-10-24 at 19 03 01](https://github.com/user-attachments/assets/780474f4-257a-4eb3-810a-39d9827453c4)

Renders the uploaded images.